kmr_mg-ud-test.conllu

```diff
@@ -9442,8 +9442,8 @@
 6	gelek	gelek	DET	det	Number=Plur|PronType=Ind	7	det	_	_
 7	Spor	spor	NOUN	n	Case=Nom|Definite=Def|Gender=Fem|Number=Plur	10	nsubj	_	_
 8	hatin	hatin	AUX	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	10	aux	_	_
-9	pêk	pêk	ADV	adv	_	10	compound:lvc	_	_
-10	anîn	anîn	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No
+9	pêk  	pêk anîn 	VERB	vblex	Typo=Yes|Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	        root	    _	        SpaceAfter=No|CorrectForm=pêkanîn
+10	anîn	_           X       _	    _                                                               9           goeswith    9:goeswith  CorrectForm=_|XML=</sic> 	                                                             _	_	    _	_
```

Comment:
@dan-zeman Could you check and specify if there is anything wrong. The goeswith doc does not cover every column.